### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by OverSx**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @BadFatRat

### CODE REVIEW

File: ConsoleApp1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            string R1 = Console.ReadLine();
+            float r1, r2, S1, S2, S3;
+            r1 = CheckNumber(R1);
+            Console.Write("Введите радиус второго круга. R2 = ");
+            string R2 = Console.ReadLine();
+            r2 = CheckNumber(R2);
+            if (r1 <= r2)
+            {
+                Console.WriteLine("R1 должен быть больше чем R2.");
+                Environment.Exit(0);
```
Comment (kryksyh) [inline ConsoleApp1/Program.cs:26]: Как я говорил на лекции, программа завершается тогда, когда завершается функция Main. 
Тут не нужны лишние сущности Environment.Exit, достаточно просто поставить return;

Comment (d-makarenko) [inline ConsoleApp1/Program.cs:26]: Как я уже говорил на лекции, программа завершается тогда, когда завершается функция Main. Тут не нужны лишние сущности Environment.Exit, достаточно просто написать return.

File: ConsoleApp1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            if (float.TryParse(user_input, out float value))
+            {
+                if (value > 0)
+                {
+                    return value;
+                }
+            }
+            Console.WriteLine("Введённая вами строка не является нужным числом. Конец программы.");
+            Environment.Exit(0);
+            return 0;
```
Comment (d-makarenko) [inline ConsoleApp1/Program.cs:46]: А тут, соответственно до `return 0;` дело никогда не дойдет, потому что программа завершится на Exit(0)

**Comment by d-makarenko** [comment]
Еще одно замечание - файлы выложены в гитхаб через бразуер, и залиты лишние файлы, экзешники и отладочные билды. Так делать не принято.

**Comment by d-makarenko** [comment]
Оценка: 4/4

**Pull request merged**